phases/phase-1/challenges/accept-referral.md

```diff
@@ -0,0 +1,28 @@
+# Accept a referral
+:::tip Note  
+This challenge is only available top users who have been referred from another user via the [Referral challenge](refer.md) and who have already created a post via the [Post challenge](create-post.md). 
+
+Upon completing this challenge, you will be rewarded with **50 Desmos Tokens**.   
+:::
+
+Once you have been properly being referred via the [Referral challenge](refer.md) and you have create a post via the [Post challenge](create-post.md), you are now ready to get 50 Desmos Tokens by accepting the referral. To do so, you need to:
```

Comment:
Once you have been properly being referred via the [Referral challenge](refer.md) and you have create a post via the [Post challenge](create-post.md), you are now ready to get 50 Desmos Tokens by accepting the referral. To do so, you need to:

->

Once you have met the above conditions and completed the challenge, you are now ready to get 50 Desmos Tokens. To do so, you need to: